src/concepts/analysis_gotchas.md

```diff
@@ -24,9 +24,10 @@ Especially severe problems with production data are announced on the `fx-data-de
 
 When you start to evaluate trends, be aware of events from the past that may invite comparisons with history. Here are a few to keep in mind:
 
-- **May 15 2022** - [Fixed potential under report `search_with_ads`][bug 1673868].
+- **May 24 - Jun10, 2022** - [`search_with_ads` drop on Firefox Desktop globally]. Upon investigation [here](https://docs.google.com/document/d/1wdU1O6Anmqs87PdyYXympTXHznoskU6pVdSgTS6ilpA/edit), it's believed most likely related to Google's core algorithm update in May 2022. 
```

Comment:
may reframe it: Upon [investigation](https://docs.google.com/document/d/1wdU1O6Anmqs87PdyYXympTXHznoskU6pVdSgTS6ilpA/edit), the issue is believed to be related to Google's core algorithm update in May 2022.